### ISSUE TITLE
iframes sometimes not working

### ISSUE THREAD

**Issue by ascii78** [typora/typora-issues#3078]
I'm including more web resources lately via the iframe block, sometimes this is not completely working, and resulting in a grey block.

Working (cicuitjs simulation)
`<iframe width="852" height="480" src="http://tinyurl.com/y2tcxz44" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>`

Non working (geogebra math figure)
`<iframe src="http://www.geogebra.org/material/iframe/id/d366bnbu/width/900/height/870/border/888888/sfsb/true/smb/false/stb/false/stbh/false/ai/false/asb/false/sri/false/rc/false/ld/false/sdz/false/ctl/false" width="900px" height="870px" style="border:0px;"> </iframe>`

This loads very quickly, shows the geogebra splash sites and then goes to grey.

![image](https://user-images.githubusercontent.com/1138654/70347950-e91b5b80-1861-11ea-91d1-fcf00272a242.png)

Not sure if it's server side or in typora.

**Comment by abnerlee** [comment]
```
<iframe src="http://www.geogebra.org/material/iframe/id/d366bnbu/width/900/height/870/border/888888/sfsb/true/smb/false/stb/false/stbh/false/ai/false/asb/false/sri/false/rc/false/ld/false/sdz/false/ctl/false" width="900px" height="870px" style="border:0px;"> </iframe>
```

This is working in my test, what's your OS/Typora version, do you have proxy set up?

**Comment by ascii78** [comment]
windows 10/1909
typora 0.9.81
no proxy

In devtools i noticed that geogebra library is loaded (it also displays the loading screen a fraction of a second before greying out)

![image](https://user-images.githubusercontent.com/1138654/70922586-a0645f00-2026-11ea-995d-089767a3ccfb.png)

**Comment by ascii78** [comment]
Just a small update, on the same network, macOS works but both Linux (Arch) and Windows 10 show the grey screen.

**Comment by ascii78** [comment]
fixed on linux/windows 0.9.87
